### DISCUSSION TITLE
Better CSS for restrictions

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#10] [Ideas]
Probably subjective, but I think restrictions in the square brackets could look better:
![image](https://github.com/stephenmk/Jitendex/assets/146892521/4b241582-6820-437d-8f3a-c16c4cd07462)

At least to me they don't feel very well "integrated" and they should stand out more

**Comment by stephenmk** [comment]
I guess I could make the brackets red. I'm open to other ideas.

![restrictions](https://github.com/stephenmk/Jitendex/assets/8003332/cbee7fbf-9bad-4fe8-86f6-fd62ce70927a)

**Comment by parfait8566** [comment]
> I guess I could make the brackets red. I'm open to other ideas.
> 
> ![restrictions](https://user-images.githubusercontent.com/8003332/279825608-cbee7fbf-9bad-4fe8-86f6-fd62ce70927a.png)

Already looks better, though I can't help but think the brackets feel a bit out of place (would still look worse without them). Maybe it's just me

**Reply by stephenmk** [reply]
I could put it in a rounded rectangle like the other tags but that's really only viable on the MDict version. For yomichan I'd have to generate thousands of SVG images on every build. Not sure what kind of effect that would have on performance in yomichan either.

If we want the restrictions tags to match the other tags, we have to decrease the font size too. That might make it difficult to read if they contain kanji.

![restrictions2](https://github.com/stephenmk/Jitendex/assets/8003332/9b20cd83-1bc1-421d-9dc5-158b6f65a92d)

**Reply by parfait8566** [reply]
Then I guess the red brackets do the job fine

**Reply by stephenmk** [reply]
I added the red square brackets in the newest version (v1.6). I guess I'll leave this discussion open in case anyone else has any ideas.
